life_server/Functions/Systems/fn_terrainSort.sqf

```diff
@@ -12,7 +12,7 @@ if !(params[["_dictionary",[],[[],0,""]]]) exitWith {};
 scopeName "main";
 
 {
-    if (worldName isEqualTo _x select 0) then {
+    if (worldName == _x select 0) then {
```

Comment:
How come the change to the `==` comparison?
`isEqualTo` is faster although case sensitive, I do not believe this has caused an issue as long as people are aware of this

Reply:
there was an error with IsEqualTo

Reply:
What error exactly? If people fill in the worldname correctly, its fine?

Reply:
Error in expression <e "main";

{
if (worldName isEqualTo _x select 0) then {
(_x select 1) breakOut >
  Error position: <select 0) then {
(_x select 1) breakOut >
  Error select: Type Bool, expected Array,String,Config entry
File \life_server\Functions\Systems\fn_terrainSort.sqf [TON_fnc_terrainSort], line 15